CONTRIBUTING.md

```diff
@@ -64,20 +76,55 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the WG for resolution and that discussion and
 compromise among collaborators be the default resolution mechanism.
 
-# Becoming a Member and Collaborator
+Collaborators are expected to follow this policy and continue to send pull requests, go through proper review, and have other collaborators merge their pull requests.
+
+### Becoming a Contributor
+
+To begin contributing to working groups and initiatives, start participating!
+
+Attend the meetings of the working group or initiative, investigate issues tagged
+as `good first issue`, open issues and pull requests, or provide insight via GitHub.
+
+Individuals can open an issue asking to be an Observer of CommComm meetings. You
+can find a great example of such an issue [here](https://github.com/nodejs/community-committee/issues/142)!
+
+### Becoming a Collaborator
+
+CommComm Collaborators are active contributors with write access to the repo(s)
+relevant to their initiative or working group.
+
+Active Contributors of a CommComm initiative or working group can be nominated by
+another Collaborator to become a Collaborator of that initiative or working group.
+On approval by the consensus process, the Contributor becomes a Collaborator.
+
+Typical activities of a Collaborator include:
+
+* Helping users and novice contributors
+* Contributing code, non-code, and documentation changes that improve the project
+* Reviewing and commenting on issues and pull requests
+* Participating in working groups or initiatives
+* Merging pull requests
+
+The CommComm periodically reviews the Collaborator list to identify inactive Collaborators.
+Past Collaborators are typically given Emeritus status. Emeriti may request that
+the CommComm restore them to active status.
+
+### Becoming a Member
+
+CommComm Members are administrative and help to remove barriers for initiatives
+and working groups under the CommComm’s scope.
 
-To become a member of the Community Committee, start participating! Attend the bi-weekly meetings, investigate issues tagged as `good first issue`,
-file issues and pull requests, and provide insight via GitHub. Request to become an Observer by
-filing an issue. Once added as an Observer to meetings, we will track attendance and participation
-for 3 months, as according to our [governance guidelines](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md#section-4-establishment-of-the-community-committee). If you meet the minimum
-attendance and are participating, the CommComm will vote to add you as a member.
+Active Collaborators of a CommComm initiative or working group can be nominated by
+a CommComm Member to become a Member of CommComm. On approval by the consensus process,
+the Collaborator becomes a Member.
 
```

Comment:
The challenge I see with self-nomination is you really want to avoid cases where the answer is 'no'.  Nomination by existing CommComm members means that most likely the answer will be 'yes'.  People can make it clear they would like to be nominated if they have not been already but it seems like having some support (enough for a nomination) will help to avoid the 'no' case.  Instead, you can get more informal feedback and suggestions on what to do in order to get to the point where a member feels comfortable nominating you.